livre/core/data/VolumeDataSourcePlugin.cpp

```diff
@@ -26,14 +26,23 @@ namespace livre
 VolumeDataSourcePlugin::VolumeDataSourcePlugin()
 {}
 
-ConstLODNodePtr VolumeDataSourcePlugin::getNode( const NodeId& nodeId ) const
+LODNode VolumeDataSourcePlugin::getNode( const NodeId& nodeId ) const
```

Comment:
if you request an invalid node the returned node should be invalid. Either you have static invalid lod node or return a copy. I did not wanted to the second because LODNode copies are not expensive.